src/csvcubed/utils/sparql_handler/column_component_info.py

```diff
@@ -0,0 +1,22 @@
+"""
+ColumnComponentInfo
+-------------------
+
+Provides access to mapping between CSV columns.
```

Comment:
I don't think this is quite right. It groups together the column with its associated component type and possibly the component.